packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -190,6 +190,8 @@ type RefNodes = { [name: string]: Element };
 const refsCache: WeakMap<RefVNodes, RefNodes> = new WeakMap();
 
 export interface LightningElement extends HTMLElementTheGoodParts, AccessibleElementProperties {
+    constructor: LightningElementConstructor;
+    shadowRoot: ShadowRoot | null;
```

Comment:
Removing the `@ts-ignore` from the prototype assignment revealed that these are missing.